### PR TITLE
Issue #166: Added format string to @Parameters annotation in Scrabble…

### PR DESCRIPTION
**Pull request by FridaTveit**
…ScoreTest. This should fix issue #166 .

### CODE REVIEW

**Comment by FridaTveit** [comment]
Did the same change for AtbashTest, BinaryTest, OctalTest, RaindropsTest, RomanNumeralsTest and PigLatinTest. This should fix issue #165, #164 , #163, #162 , #161 and #160 .

**Comment by FridaTveit** [comment]
Sure @jtigger, will do! :)

File: exercises/binary/src/test/java/BinaryTest.java [hunk 1]
```diff
@@ -12,10 +12,11 @@
 @RunWith(Parameterized.class)
 public class BinaryTest {
 
-    private String input;
-    private int expectedOutput;
+    private String binaryNumberAsString;
+    private int decimalNumber;
 
-    @Parameters
+    @Parameters(name = "{index}: expected result {1} when converting binary number " +
```
Comment (jtigger) [inline exercises/binary/src/test/java/BinaryTest.java]: "The fewer words written, the more of them read."  Do you think we can trim up this sentence?

File: exercises/binary/src/test/java/BinaryTest.java [hunk 2]
```diff
@@ -34,16 +35,16 @@
         });
     }
 
-    public BinaryTest(String input, int expectedOutput) {
-        this.input = input;
-        this.expectedOutput = expectedOutput;
+    public BinaryTest(String binaryNumberAsString, int decimalNumber) {
```
Comment (jtigger) [inline exercises/binary/src/test/java/BinaryTest.java]: 👍

File: exercises/octal/src/test/java/OctalTest.java [hunk 3]
```diff
@@ -11,10 +11,11 @@
 @RunWith(Parameterized.class)
 public class OctalTest {
 
-    private String input;
-    private int expectedOutput;
+    private String octalNumberAsString;
+    private int decimalNumber;
 
-    @Parameterized.Parameters
+    @Parameterized.Parameters(name = "{index}: expected result {1} when converting octal number " +
```
Comment (jtigger) [inline exercises/octal/src/test/java/OctalTest.java]: Same comment about simplifying this sentence as above.

File: exercises/pig-latin/src/test/java/PigLatinTest.java [hunk 4]
```diff
@@ -11,10 +11,10 @@
 @RunWith(Parameterized.class)
 public class PigLatinTest {
 
-    private String input;
-    private String expectedOutput;
+    private String englishPhrase;
+    private String pigLatinTranslation;
 
-    @Parameterized.Parameters
+    @Parameterized.Parameters(name = "{index}: expected \"{0}\" to translate to the pig latin phrase \"{1}\"")
```
Comment (jtigger) [inline exercises/pig-latin/src/test/java/PigLatinTest.java]: 👍

File: exercises/binary/src/test/java/BinaryTest.java (hunk 1 above)
Comment (FridaTveit) [inline exercises/binary/src/test/java/BinaryTest.java]: I've changed it :)

File: exercises/octal/src/test/java/OctalTest.java (hunk 3 above)
Comment (FridaTveit) [inline exercises/octal/src/test/java/OctalTest.java]: I've changed it :)

**Comment by jtigger** [comment]
Thanks for your time and attention on this @FridaTveit!  Looks great!

**Pull request merged**